src/web/app/pom.xml

```diff
@@ -1678,6 +1678,11 @@
           <artifactId>gs-dggs-clickhouse</artifactId>
           <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I wish we didn't have to do this.

Reply:
Actually we want to do more eventually, that is, split the ogcapi module in smaller parts, allowing to bring in only one of the services. Something that @jodygarnett wanted to work on AFAIK.

Reply:
I mean having to manually add the module as a dependency to the web `pom.xml`. Its not an issue for this PR, just a general issue.

Reply:
When developing I normally start GeoServer from the IDE, and control which extensions/community modules using profiles... that's pretty handy. I normally keep the list of deps included small (the set that I need plus a few common ones), otherwise the server gets heavier and some extensions are not compatible with each other.

All of this is controlled by profiles and dependencies in them.
What would be a better alternative to control what gets included?